content/concepts/transport.md

```diff
@@ -210,3 +210,76 @@ Following the multiaddress format described earlier, a standard QUIC connection
 look like: `/ip4/127.0.0.1/udp/65432/quic/`.
 
 In this section, we offered an overview of QUIC and how QUIC works in libp2p.
+
+## WebTransport
+
+Another transport protocol under development at the IETF is WebTransport.
+WebTransport is a new specification that uses QUIC to offer an alternative to
+WebSockets. It can be considered WebSockets over QUIC by allowing 
+browsers to establish a stream-multiplexed and bidirectional 
+connection to servers. 
+
+The specification can depend on and reuse the QUIC infrastructure in place 
+to offer WebSockets that take the benefits of UDP and offer streams without 
+head-of-line blocking.
```

Comment:
```suggestion
```

Instead, can we explain why WebTransport allows us to connect _any_ libp2p browser node with _any_ server node? This is because it supports verifying the certificate by its certificate hash (https://www.w3.org/TR/webtransport/#dom-webtransportoptions-servercertificatehashes), whereas for WebSocket is was necessary that the TLS cert was signed by a trusted CA.

Reply:
Can we remove this sentence?